app/services/hood/onboarder.rb

```diff
@@ -1,6 +1,6 @@
 class Hood
   class Onboarder
-    RequiredHeaders = %i(
+    RequiredFields = %i(
```

Comment:
Converted this to be more generic and support an array of hashes (which could come from parsing a CSV into that, or a JSON object into that).